### PR TITLE
improved: marzneshin feature list

### PR DESCRIPTION
**Pull request by Krr0ptioN**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,38 +56,53 @@
...
+hosts, users, services entites. While providing monitoring and system 
+overview statistics.
 
-## How it works 
+On another hand, the system (backend) manages traffic distribution, v2ray
+configurations, proxies and VPN backends, users subscription to proxies, etc. 
 
-Marzneshin controls the nodes connected to it; monitoring/disabling/enabling users on marznode instances
-while marznode manages vpn backends (such as xray)
+Marzneshin controls the [Marznodes](https://github.com/khodedawsh/marznode)
```
Comment (khodedawsh) [inline README.md:71]: no the

File: README.md [hunk 2]
```diff
@@ -379,12 +376,7 @@ We ❤️‍🔥 contributors! If you'd like to contribute, please check out our
 Check [open issues](https://github.com/khodedawsh/marzneshin/issues) to help the progress of this project.
 
 <p align="center">
-Thanks to the all contributors who have helped improved Marzban & Marzneshin:
-</p>
-<p align="center">
-<a href="https://github.com/Gozargah/Marzban/graphs/contributors">
-  <img src="https://contrib.rocks/image?repo=Gozargah/Marzban" />
-</a>
+Thanks to the all contributors who have helped Marzneshin:
```
Comment (khodedawsh) [inline README.md:379]: no the

File: README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <h1 align="center"/>Marzneshin</h1>
 
 <p align="center">
-    A Fork of Marzban aiming for scalability
+    An Scalable and Comprehensive Proxy Engine management panel.
```
Comment (khodedawsh) [inline README.md:4]: A scalable*

**Review by khodedawsh — CHANGES_REQUESTED** [review]

**Review by Krr0ptioN — COMMENTED** [review]
Good

**Review by khodedawsh — APPROVED** [review]

**Pull request merged**